scripts/devel/build_and_install.py

```diff
@@ -77,7 +117,21 @@ def main(argv=sys.argv[1:]):
         if args.clean_after:
             clean_workspace(args.workspace_root)
 
-    return rc
+    if not args.run_abichecker:
```

Comment:
If the ABI checker runs then the possibly nonzero return code from the build is discarded.
Do we need to check that before continuing?

Reply:
I think you are right. f0161fe7bc4b4535e1844dfe1910c7048303af59